Cargo.toml

```diff
@@ -28,4 +28,5 @@ log = "0.4.17"
 zstd = "0.12.3"
 
 [dev-dependencies]
+cargo2junit = "0.1.12"
```

Comment:
We still have `cargo install` in the build script - we could just add `--version 0.1.12` there and remove it from here I think